ios/browser/api/url_sanitizer/BUILD.gn

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+source_set("url_sanitizer") {
+  configs += [ "//build/config/compiler:enable_arc" ]
+  sources = [
+    "url_sanitizer_service+private.h",
+    "url_sanitizer_service.h",
+    "url_sanitizer_service.mm",
+  ]
+  deps = [
+    "//base",
+    "//brave/components/url_sanitizer/browser",
+    "//net",
+    "//url",
+  ]
+}
```

Comment:
this needs `frameworks = [ "Foundation.framework" ]` for https://github.com/brave/brave-core/pull/19079/files#diff-9308d1b53e0e1c346b56bf5e41448ca7c6ca2c34f80b8455457921a36361e4fbR9

Reply:
Done